### PR TITLE
[16.0][NEW] pos_order_to_sale_order_report: Add new module - download report after sale order creation from POS

### PR DESCRIPTION
**Pull request by GabbasovDinar**
This module extends the functionality of the pos_order_to_sale_order module which allows download a report from Point of Sale after creating a sales order.

### CODE REVIEW

File: pos_order_to_sale_order_report/static/src/js/Popups/CreateOrderPopup.esm.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import CreateOrderPopup from "point_of_sale.CreateOrderPopup";
+import Registries from "point_of_sale.Registries";
+
+const PosDeliveryCarrierCreateOrderPopup = (CreateOrderPopup) =>
```
Comment (legalsylvain) [inline pos_order_to_sale_order_report/static/src/js/Popups/CreateOrderPopup.esm.js]: Why this name ? PosDeliveryCarrierCreateOrderPopup

Comment (GabbasovDinar) [reply pos_order_to_sale_order_report/static/src/js/Popups/CreateOrderPopup.esm.js]: ooops, sorry, the name was accidentally copied from another module, I corrected it

**Review by ivs-cetmix — APPROVED** [review]
Functional review OK

**Comment by ivs-cetmix** [comment]
hey @legalsylvain  fixed,  would appreciate your review when possible

**Review by legalsylvain — APPROVED** [review]
Lgtm. Thanks !

**Pull request merged**